docs/content/reference/video.md

```diff
@@ -0,0 +1,73 @@
+* Compressed as a single [`AssetVideo`](../reference/types/archetypes/asset_video.md), using e.g. MP4 and AV1 or H.264.
+
+These alternatives range on a scale of "simple, lossless, and big" to "complex, lossy, and small".
+
+If you want lossless encoded images (with no compression artifacts), then you should log each video frame as `Image`.
+This will use up a lot of space and bandwidth. You can also encode them as PNG and log them as `EncodedImage`,
+though it should be noted that PNG encoding usually does very little for the file size of photographic images.
+
+If you want to reduce bandwidth and storage cost, you can encode each frame as a JPEG and log it using `EncodedImage`. This can easily reduce the file sizes by almost two orders of magnitude with minimal perceptiall loss.
+This is also very simple to do, and the Python logging SDK has built-in support for it using [`Image.compress`](https://ref.rerun.io/docs/python/0.18.2/common/archetypes/#rerun.archetypes.Image.compress).
+
+Finally, you can encode the images as a video file, and log it using `AssetVideo`.
+This gives the best compression ratio, reducing file sizes and bandwidth requirements.
+
+## `AssetVideo` limitations
+Video support is new in Rerun, and has several limitations:
+
+* Video playback only works in the web viewer
+* There is no video encoder in the Rerun SDK, so you need to create the video file yourself
+* Only the MP4 container format is supported
+* Only a limited sets of codecs are supported (see below)
+* There is no audio support
+
+## Web viewer support
+As of writing, playback of `AssetVideo` is only supported on the web viewer.
+Native video playback is coming, and can be tracked [in this GitHub issue](https://github.com/rerun-io/rerun/issues/7298).
+
+Video playback is done using the browsers own video decoder, so the supported codecs depends on your browser.
+
+Overall, we recommend using Chrome or another Chromium-based browser, as it seem to have the best video support as of writing.
+
+When choosing codec, we recommend [AV1](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Video_codecs#av1), as that seems to have the best overall playback support. Since AV1 is patent-free, it is also likely the first codec we will support in the native viewer.
+
+For decoding video in the Web Viewer, we use the [WebCodecs API](https://developer.mozilla.org/en-US/docs/Web/API/WebCodecs_API).
+This API enables us to take advantage of the browser's hardware accelerated video decoding capabilities.
+It is implemented by all modern browsers, but with varying levels of support for different codecs, and varying levels of quality.
+
+With that in mind, here are the browsers which we have tested and verified to work:
+
+|            | Linux  | macOS | Windows |
+| ---------- | ------ | ----- | ------- |
+| Firefox    | ✅[^2] | ✅    | ✅      |
+| Chrome[^1] | ✅     | ✅    | ✅      |
```

Comment:
Windows needs a footnote that lines up with the issues described in the troubleshooting guide in https://github.com/rerun-io/rerun/pull/7537